scripts/splitSources.py

```diff
@@ -18,7 +18,7 @@ def uncaught_exception_hook(exc_type, exc_value, exc_traceback):
     # The script `scripts/ASTImportTest.sh` will interpret return code 3
     # as a critical error (because of the uncaught exception) and will
     # terminate further execution.
-    print("Unhandled exception: %s", "".join(traceback.format_exception(exc_type, exc_value, exc_traceback)))
+    print("Unhandled exception:" + "".join(traceback.format_exception(exc_type, exc_value, exc_traceback)))
```

Comment:
```suggestion
    print("Unhandled exception: " + "".join(traceback.format_exception(exc_type, exc_value, exc_traceback)))
```
Otherwise this will put the trace directly next to the `:`. :)